packages/frontend/app/routes/assets.$assetId.tsx

```diff
@@ -119,7 +139,63 @@ export default function ViewAssetPage() {
             </div>
           </div>
         </div>
-        {/* Peer Liquidity Info - END */}
+        {/* Asset Liquidity Info - END */}
+        {/* Asset Fee Info */}
+        <div className='grid grid-cols-1 py-3 gap-6 md:grid-cols-3 border-b border-pearl'>
+          <div className='col-span-1 pt-3'>
+            <h3 className='text-lg font-medium'>Sending Fee</h3>
+            {asset.sendingFee ? (
+              <p className='text-sm'>Created at {asset.sendingFee.createdAt}</p>
+            ) : null}
+            <ErrorPanel errors={response?.errors.sendingFee.message} />
+          </div>
+          <div className='md:col-span-2 bg-white rounded-md shadow-md'>
+            <div className='flex justify-end p-4'>
+              <Button
+                aria-label='view asset fees page'
+                type='button'
+                to={`/assets/view-fees/${asset.id}`}
+              >
+                View all historic fees
+              </Button>
+            </div>
+            <Form method='post' replace preventScrollReset>
+              <fieldset disabled={currentPageAction}>
+                <div className='w-full p-4 space-y-3'>
+                  <Input type='hidden' name='assetId' value={asset.id} />
+                  <Input
+                    type='number'
+                    name='fixed'
+                    label='Fixed Fee'
+                    defaultValue={asset.sendingFee?.fixed ?? undefined}
+                    error={response?.errors.sendingFee.fieldErrors.fixed}
+                  />
+                  <Input
+                    type='number'
+                    name='basisPoints'
+                    label='Basis Points'
+                    defaultValue={asset.sendingFee?.basisPoints ?? undefined}
+                    error={response?.errors.sendingFee.fieldErrors.basisPoints}
+                  />
+                  <p className='text-gray-500 text-sm mt-2'>
+                    A single basis point is equal to 0.01% of the total fee.
```

Comment:
maybe a stretch, but it could also be nice to make it response to user input. For example they type in 50 and see "50 basis points is equal to 0.50% of the total fee" (or the worked example Sabine is suggesting).

Reply:
How about "A single basis point is a fee equal to 0.01% of the total amount. A fee of 1 basis point on $100 is $0.01."

Reply:
That works. Maybe we should create an issue to make this line adapt like @BlairCurrey suggested. That could even be a last minute hacktoberfest contribution.